sim/battle.ts

```diff
@@ -175,17 +193,10 @@ export class Battle extends Dex.ModdedDex {
 				if (hasEventHandler) this.addPseudoWeather(rule);
 			}
 		}
-		if (options.p1) {
-			this.setPlayer('p1', options.p1);
-		}
-		if (options.p2) {
-			this.setPlayer('p2', options.p2);
-		}
-		if (options.p3) {
-			this.setPlayer('p3', options.p3);
-		}
-		if (options.p4) {
-			this.setPlayer('p4', options.p4);
+		for (const side of SIDE_IDS) {
```

Comment:
I don't really like the `SIDE_IDS`, especially not imported from `sim/side`, and especially not unnamespaced.

Reply:
Considering it's only used here, it might be better as a literal.

Reply:
is `Side.IDS` any better?

Reply:
It is, but I'm still partial to the array literal.

Reply:
*d' accord. je m'en fiche*